### PR TITLE
NO-ISSUE: adding wait for arm instance

### PR DESCRIPTION
**Pull request by eifrach**

### CODE REVIEW

File: ansible_files/roles/equinix/heterogeneous_create_infra/tasks/wait_for_instance.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+    - name: Download Metal cli
+      ansible.builtin.get_url:
+        url: "https://github.com/equinix/metal-cli/releases/download/v{{ metal_version }}/metal-linux-amd64"
+        dest: /tmp/metal
+        mode: '0775'
+        checksum: "sha256:https://github.com/equinix/metal-cli/releases/download/v{{ metal_version }}/metal-cli_{{ metal_version }}_checksums.txt"
+    
+    - name: "Wait for ARM instance {{ item }}"
```
Comment (adriengentil) [inline ansible_files/roles/equinix/heterogeneous_create_infra/tasks/wait_for_instance.yaml:8]:
```suggestion
    - name: "Wait for plans to be available"
```
`{{ item }}` doesn't show properly:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_assisted-test-infra/2313/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-day2-arm-workers/1727238404069920768/artifacts/e2e-metal-assisted-day2-arm-workers/assisted-equinix-setup/build-log.txt

File: ansible_files/roles/equinix/heterogeneous_create_infra/tasks/wait_for_instance.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      shell:
+        cmd: /tmp/metal capacity get -f  -P "{{ item }}"  | grep 'normal\|limited' | wc -l
+      register: metal_cmd
+      until: metal_cmd.stdout != 1
```
[the remark this answers is not in the document]
Comment (eifrach) [reply ansible_files/roles/equinix/heterogeneous_create_infra/tasks/wait_for_instance.yaml:12]: no it not causing an issue it's python know's to handle it  
but that should be `>  0`  

I will work on it

**Review by adriengentil — APPROVED** [review]

**Comment by adriengentil** [comment]
/override ci/prow/e2e-metal-assisted ci/prow/e2e-metal-assisted-kube-api-net-suite

**Pull request merged**